### ISSUE TITLE
Click menu "View->Other Windows->Python Environment" and create a new one by pointing to a local installed Python like 3.7.0 - this step is not correct since Visual Studio 2015 doesn't support Python greater than 3.5. 

### ISSUE THREAD

**Issue by lululilliancoding** [Azure/azure-cli#14655]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
Under Visual Studio 2015 tag, 
Click menu "View->Other Windows->Python Environment" and create a new one by pointing to a local installed Python like 3.7.0 - this step is not correct since Visual Studio 2015 doesn't support Python greater than 3.5. 

**To Reproduce**

**Expected behavior**
Make description correct. 

**Environment summary**

**Additional context**

**Comment by yungezz** [comment]
hi @lululilliancoding are you using Azure CLI? seems the issue is VisualStudio, not azure cli related. Will triage later when have more info.

**Comment by lululilliancoding** [comment]
I am using VS 2015 but the document said suggest use python 3.7 well VS 2015 doesn't support python 3.7

**Comment by yungezz** [comment]
@lululilliancoding thanks for the info. unfortunately, this repo is for Azure CLI tool. It's not the right place for VS related issue. could  you pls create issue at VS? thanks.

**Comment by lululilliancoding** [comment]
Yes it is VS2015 don't support python 3.7 and you can say it is VS issue rather than Azure CLI tool issue but the documentation/description is on Azure CLI repo right? I mean the description is wrong.

**Comment by lululilliancoding** [comment]
I heart from other dev that how they point VS to python 3.7+ is they install both VS 2015 and VS 2019 then create virtual env via VS 2019 so since it can point to python 3.7+ (VS 2015 can't) then go back to use VS 2015 to run tests. I mean the steps description is not updated for new developers who want to develop CLI cmdlets.
